src/aqua/installation-spell/src/aqua/upload.aqua

```diff
@@ -1,8 +1,13 @@
+aqua Upload declares *
```

Comment:
A bit unrelated question: why do you choose the `aqua` keyword, which doesn't have any meaning, and not `lib`, which is exactly what this is? 
imho this looks more understandable the original
```aqua
lib Upload declares *
```